graphics/QtPBFImagePlugin/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        tumic0 QtPBFImagePlugin 1.1
+categories          graphics
+platforms           darwin
+license             LGPL-3
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         PBF image plugin for Qt5
+long_description    Qt image plugin for displaying Mapbox vector tiles.
+
+homepage            https://github.com/tumic0/QtPBFImagePlugin
```

Comment:
The explicit `homepage` line can be removed because the GitHub portgroup will set it to this.

Reply:
Removed.